sites/example-project/src/pages/text-and-metrics/markdown/+page.md

```diff
@@ -1,3 +1,5 @@
+<script> import { base } from "$app/paths"; </script>
```

Comment:
So if I understand, if people wants `links` in their reports they will have to import `base`?

Reply:
It was added to preprocess so it shouldn't be required, just needed in the example project